_database-integrations/dynamodb/dynamodb-v1.md

```diff
@@ -0,0 +1,183 @@
+  - item: |
+      **Files that adhere to Stitch's file requirements**:
+
+      {{ site.data.taps.extraction.file-systems.file-requirements.support-table | flatify }}
+
+
+
+# -------------------------- #
+#     Setup Instructions     #
+# -------------------------- #
+
+setup-steps:
+  - title: "Retrieve your Amazon Web Services account ID"
+    anchor: "retrieve-aws-account-id"
+    content: |
+      {% include integrations/shared-setup/aws-s3-iam-setup.html type="retrieve-account-id" %}
+      
+  - title: "Add {{ integration.display_name }} as a Stitch data source"
+    anchor: "add-stitch-data-source"
+    content: |
+      {% include shared/database-connection-settings.html type="general" %}
+
+
+  - title: "Define the historical sync"
+    anchor: "define-historical-sync"
+    content: |
+      {% include integrations/shared-setup/configure-table-settings-file-server.html type="define-historical-sync" %}
+
+  - title: "Create a replication schedule"
+    anchor: "create-replication-schedule"
+    content: |
+      {% include integrations/shared-setup/replication-frequency.html %}
+
+  - title: "Grant access to your bucket using AWS IAM"
+    anchor: "grant-access-bucket-iam"
+    content: |
+      {% include integrations/shared-setup/aws-s3-iam-setup.html type="aws-iam-access-intro" %}
+
+    substeps:
+      - title: "Create an IAM policy"
+        anchor: "create-iam-policy"
+        content: |
+          {% include integrations/shared-setup/aws-s3-iam-setup.html type="create-iam-policy" %}
+          
+      - title: "Create an IAM role for Stitch"
+        anchor: "create-stitch-iam-role"
+        content: |
+          {% include integrations/shared-setup/aws-s3-iam-setup.html type="create-stitch-iam-role" %}
+
+      - title: "Check and save the connection in Stitch"
+        anchor: "check-save-stitch-connection"
+        content: |
+          {% include integrations/shared-setup/aws-s3-iam-setup.html type="check-and-save" %}
+
+  - title: "Select data to replicate"
+    anchor: "setting-data-to-replicate"
+    content: |
+      {% include integrations/databases/setup/syncing.html %}
+
+
+# -------------------------- #
+#      Replication Info      #
+# -------------------------- #
+
+replication-sections:
+  - title: "Log-based Incremental Replication"
+    anchor: "log-based-incremental-replication"
+    summary: "Details about Log-based Incremental Replication via DynamoDB streams."
+    content: |
+      This integration supports Full Table and Log-based replication. Log-based replication is done through DynamoDB streams, not Amazon Kinesis streams. In order to use streams, they must be enabled on the tables created in DynamoDB. When in your AWS account, go to the `manage streams` section and choose `New Image` or `New and old images`. If one of these aren't selected, the integration will fail during sync. This is handled on a table-by-table basis, so you must enable streams on each.
```

Comment:
All of this is for the first paragraph:

> This integration supports Full Table and Log-based replication.

I'd remove this sentence entirely - you're only talking about Log-based here, so no need to mention Full.

---

> Log-based replication is done through DynamoDB streams, not Amazon Kinesis streams.

Try to relate this to the integration itself, and include links to the documentation about the feature:. Ex:

```
Stitch's DynamoDB integration uses [DynamoDB Streams](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.html) to perform Log-based Incremental Replication.
```

---

> In order to use streams, they must be enabled on the tables created in DynamoDB.

While true, the phrasing isn't entirely accurate as it relates to the integration. **In order to use Log-based Replication**, streams must be enabled on any table in DynamoDB the user wants to replicate using Log-based Replication. So while streams must be enabled in order to use them, here you're trying to guide the user to understanding that they're enabling them **in this case** so they can use this Replication Method.

So, perhaps: `To use Log-based Incremental Replication, streams must be enabled on every table in DynamoDB you want to replicate using this Replication Method.` This way, you can also get rid of the last sentence in this paragraph (`This is handled on a table-by-table basis, so you must enable streams on each.`) and tighten things up a bit.

---

> When in your AWS account, go to the `manage streams` section and choose `New Image` or `New and old images`. If one of these aren't selected, the integration will fail during sync.

I think this is important and should be called out a bit more - right now it gets lost in the rest of the paragraph. Here's what I'd do:

1. Make this its own paragraph
2. [Link right to the documentation for enabling streams](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.html#Streams.Enabling)(meaning you don't have to explain it now)
3. Calling out the options the integration supports. 

So, perhaps:

```
Refer to [Amazon's documentation](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.html#Streams.Enabling) for instructions on enabling streams for DynamoDB tables. The **Manage Stream** options must be one of the following, or replication will be unsuccessful:

- `New Image`
- `New and old images`
```